server/src/assets/dto/update-asset.dto.ts

```diff
@@ -1,5 +1,9 @@
 import { PartialType } from '@nestjs/mapped-types';
+import { Exclude } from 'class-transformer';
 
 import { CreateAssetDto } from './create-asset.dto';
 
-export class UpdateAssetDto extends PartialType(CreateAssetDto) {}
+export class UpdateAssetDto extends PartialType(CreateAssetDto) {
+  @Exclude()
+  searchtitle: string;
```

Comment:
I don't believe we need to exclude this here, since the return type is the `entity.asset.ts`